core/api/src/main/java/org/trellisldp/api/MementoService.java

```diff
@@ -30,29 +28,16 @@
  */
 public interface MementoService {
 
-    /**
-     * Create a new Memento for a resource.
-     * @param identifier the resource identifier
-     * @param time the time of the Memento
-     * @param data the data to save
-     * @return a new completion stage that, when the stage completes normally, indicates that Memento resource was
-     * successfully created in the corresponding persistence layer. In the case of an unsuccessful write operation,
-     * the {@link CompletableFuture} will complete exceptionally and can be handled with
-     * {@link CompletableFuture#handle}, {@link CompletableFuture#exceptionally} or similar methods.
-     */
-    CompletableFuture<Void> put(IRI identifier, Instant time, Stream<? extends Quad> data);
-
     /**
      * Create a new Memento for a resource.
      * @param resource the resource
+     * @param time the time at which this Memento should be created
```

Comment:
This isn't really very good-- what do we want to say here? The time at which the Memento is _defined_?